src/ring_theory/noetherian.lean

```diff
@@ -201,6 +201,30 @@ begin
     { exact zero_smul _ }, { exact λ _ _ _, add_smul _ _ _ } }
 end
 
+/-- The image of a finitely generated ideal is finitely generated. -/
+lemma map_fg_of_fg {R S : Type*} [comm_ring R] [comm_ring S] (I : ideal R) (h : submodule.fg I)
```

Comment:
Can't you deduce this easily from `fg_map`? (Which should maybe be called `fg.map` to enable dot notation).

Also (but not important), can you write `I.fg` instead of `submodule.fg I`? Or does that not work?

Reply:
`I.fg` works, thanks.

`fg_map` doesn't work, at least not directly: `I.fg` means finitely generated as `R`-module, but `(I.map f).fg` is as `S`-module.